composer.json

```diff
@@ -12,7 +12,7 @@
         "php": ">=7.4.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^9"
+        "phpunit/phpunit": "^9.5.25"
```

Comment:
Package versions should always defined explicit. That exclude unwanted side-effects when packages updated by creator.